core/build.gradle.kts

```diff
@@ -272,15 +273,18 @@ val integTestAll by tasks.registering() {
 }
 
 for (db in listOf("h2", "mysql", "oracle", "postgresql")) {
-    val task = tasks.register("integTest" + db.capitalize(), Test::class) {
+    val task = tasks.register("integTest" + db.replaceFirstChar { if (it.isLowerCase()) it.titlecase(
```

Comment:
Since we use the capitalize logic twice in here maybe worth refactoring the logic to a separate line.

Reply:
done